students/jemielity_kamil/lesson_03_functions/fibonacci_numbers.py

```diff
@@ -0,0 +1,19 @@
+def fib(n):
+    if n == 1:
+        return 1
+    elif n == 0:
+        return 0
+    else:
+        return fib(n - 1) + fib(n - 2)
+
+
+while True:
+    number = int(input("Positive integer: "))
+    try:
+        if number < 0:
+            raise ValueError
```

Comment:
Ok, I see that the check for negative number is made outside of fib function. It's better to move this check to fib method instead